app/views/publishproposal.html

```diff
@@ -17,5 +17,8 @@
     <h4>Date Time Published: {{publication.dateTime}}</h4>
     <button ui-sref="proposalPublication({id: publication._links.self.href.split('/').pop()})">See {{publication.commentedBy._embeddedItems.length}} comments</button>
     <button ui-sref="createComment({id: publication._links.self.href.split('/').pop()})">Add comment</button>
+    <span ng-if="role.authority == 'ROLE_COORDINATOR'">
+      <button ui-sref="registerproposal({id: publication._links.self.href.split('/').pop()})">Register proposal</button>
```

Comment:
This is taking you to the existing state to list registered proposals, it should take you to a state that creates a new registration or invoke a controller function that directly creates them